site/content/en/docs/presentations/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Presentations"
+weight: 3.5
+description: >
+  Available online presentations about minikube
+---
+
+Presentations about the minikube project, mostly from the Kubernetes blog and the KubeCon conference.
+
+## Kubernetes Blog
+
+### 2016
+
+2016-05-30, Kubernetes 1.3, Minikube 0.1.0
+
+* _This is the initial release of Minikube._
+
+2016-07-11, Kubernetes 1.3, Minikube 0.5.0
+
+* <https://kubernetes.io/blog/2016/07/minikube-easily-run-kubernetes-locally/>
+
+### 2017
```

Comment:
@afbjorklund   why empty here ? could we remove the empty years?

Reply:
Will do